src/Model/Article.vala

```diff
@@ -32,6 +32,9 @@ public class FeedReader.Article : GLib.Object {
 	private int m_lastModified;
 	private int m_pos;
 
+	private static string clock_format =
+		new GLib.Settings("org.gnome.desktop.interface").get_string("clock-format");
```

Comment:
Is this a reasonable thing to do? I don't really understand Vala's `static` semantics. It works, but I don't know if it's a bad idea.

Reply:
I'm not sure either. I would assume the settings object is created once and since there is no reference holding it, runs out of scope and gets freed.
But luckily it's quite easy to find out. Just take a look at the gerated c code ;)